doc/source/library_description/template.rst

```diff
@@ -2,21 +2,88 @@
 Library Template
 ################
 
-Within the `PyAnsys Organization <https://github.com/pyansys>`_, there
-is a `template repository <https://github.com/pyansys/template>`_
-where you can `create a repository from a template`_ and create a 
-PyAnsys project that follows the guidelines specified in this developer's
-guide.
+Starting a new project from scratch is a tedious task. For this reason, the
+`ansys-tempaltes` tool was created in order to simplify the starting process and
```

Comment:
```suggestion
`ansys-templates` tool was created in order to simplify the starting process and
```

Reply:
Good catch @MaxJPRey, solved in the incoming pushed changes.